### ISSUE TITLE
Indent is inconsistent when call expression is used as variable initial value

### ISSUE THREAD

**Issue by soda0289** [eslint/eslint#8607]
**Tell us about your environment**

* **ESLint Version:**
master
* **Node Version:**
4
* **npm Version:**
4

**What parser (default, Babel-ESLint, etc.) are you using?**
typescript

**Please show your full configuration:**

```
module.exports = {
    "env": {
        "es6": true,
        "node": true,
        "mocha": true
    },
    "parser": "typescript-eslint-parser",
    "parserOptions": {
        "sourceType": "module",
        "ecmaFeatures": {
            "jsx": true
        }
    },
    "rules": {
        "indent": [
            "error",
            4,
            {
                MemberExpression: 1,
                SwitchCase: 1,
                FunctionDeclaration: {
                    body: 1,
                    parameters: 1
                },
                FunctionExpression: {
                    body: 1,
                    parameters: 1
                },
                CallExpression: {
                    arguments: 1
                },
                VariableDeclarator: 1
            }
        ]
    }
};

```

**What did you do? Please include the actual source code causing the issue.**

```js
// Invalid
const foo = a.b(),
    longName
        = memberFunction(
            'bar'
        );
```

**What did you expect to happen?**
Should be a valid indent. If I move the equal operator to the same line as the variable identifier it is consider valid indentation.
```js
// Valid
const foo = a.b(),
    longName =
        memberFunction(
            'bar'
        );
```

**What actually happened? Please include the actual, raw output from ESLint.**
Invalid indentation.
```
/home/reyad/Workspace/eslint-test/test-indent.ts
  4:1  error  Expected indentation of 8 spaces but found 12  indent
  5:1  error  Expected indentation of 4 spaces but found 8   indent
```

**Comment by soda0289** [comment]
I'll try and create a fix. This is a test case I forgot about when ignoring the equal operator in variable declarator nodes.

**Comment by not-an-aardvark** [comment]
Seems like a bug to me.
